packages/gatsby/src/schema/__tests__/__snapshots__/build-schema.js.snap

```diff
@@ -1,10 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Built-in types includes built-in types 1`] = `
-"\\"\\"\\"Provides default value for input field.\\"\\"\\"
-directive @default(value: JSON!) on INPUT_FIELD_DEFINITION
```

Comment:
Removal of this directive comes from `graphql-compose` upgrade, right?

Was it ever possible for schema customization to make use of it? I.e. when defining custom field with args? I.e.:
```
args: {
  blah: `String @default("wat")`
}
```
?

Not that we have to keep it if above is possible, just thinking if this is potential breaking change (to document)

Reply:
Yes, it was removed here: https://github.com/graphql-compose/graphql-compose/commit/aa479327f88a0815232f7fbe9783e85c7542978e

I am not sure if we should include notes about it in our migration guide. It was not documented in Gatsby. In fact, we claimed that there is no built-in way to do this: https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#setting-default-field-values